### DISCUSSION TITLE
Wake on LAN

### DISCUSSION THREAD

**Discussion by rgreil** [gravitational/teleport#38025] [Q&A] [answered]
Hi,

is there really no Wake on LAN feature or did I just missed it?

**Comment by stevenGravy** [comment]
Not sure the scenario you have in mind but Teleport does not have current support for that. If you have a specific use case you think would enhance Teleport to have this please log a feature request. https://github.com/gravitational/teleport/issues

**Reply by rgreil** [reply, marked as answer]
Thanks. I've logged a feature request: https://github.com/gravitational/teleport/issues/38073
